NCIOCPL/cgov-digital-platform
# (#4151) Fix node_type instances

Closes #4151

## Testing Instructions

* This updates the configurations that generate URLs
   * So create a new page of each content type from the content menu. (Media are not affected)
* Also how the Contact Info blocks are created for Biographies and Press Releases.
   * On a new ODE the contact blocks should show correctly.
* There is also an update hook to modify any content blocks that were created in the old way.
   * This really can only be tested in ACSF, and it would only affect any site created after the 9.3 upgrade, so maybe nationalcancerplan.cancer.gov?
